### PR TITLE
Replaced use of copy.deepcopy()

### LINKED ISSUE #2331: Update tutorial to avoid use of copy.deepcopy()- Convolution/Batch Norm Fuser in FX
Update the  [Building a convolution/batch norm fuser in FX](https://github.com/pytorch/tutorials/blob/6ae56a1a2639308550a69c5006e879d331f05b6d/intermediate_source/fx_conv_bn_fuser.py#L107) to avoid the use of copy.deepcopy().
Tutorial link: https://github.com/pytorch/tutorials/blob/6ae56a1a2639308550a69c5006e879d331f05b6d/intermediate_source/fx_conv_bn_fuser.py#L107

See issue #2177  for more information

cc @eellison @suo @gmagogsfm @jamesr66a @msaroufim @SherlockNoMad @albanD @svekars @carljparker @NicolasHug @kit1980 @subramen

### PR DESCRIPTION
**Pull request by JoseLuisC99**
Fixes #2331

## Description
Replaces use of `copy.deepcopy()` in Convolution/Batch Norm Fuser tutorial.

### CODE REVIEW

**Comment by kit1980** [comment]
And looking close, I think the `deepcopy` is fine there, no need to change really.

**Comment by JoseLuisC99** [comment]
I'm so sorry, it's more correct to use `load_state_dict` instead of load as mentioned in [Deep copying PyTorch modules](https://discuss.pytorch.org/t/deep-copying-pytorch-modules/13514).
